ssv/spectest/tests/msgcontainer/quorum_post_consensus_msg.go

```diff
@@ -0,0 +1,27 @@
+package msgcontainer
+
+import (
+	"github.com/bloxapp/ssv-spec/ssv/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// QuorumPostConsensusMsg tests adding a quorum of post consensus message to container
+func QuorumPostConsensusMsg() tests.SpecTest {
+	ks := testingutils.Testing4SharesSet()
+
+	return &SpecTest{
+		Name: "quorum post consensus message",
+		MsgsToAdd: []*types.SignedPartialSignatureMessage{
+			testingutils.PostConsensusSyncCommitteeContributionMsg(ks.Shares[1], 1, ks),
+			testingutils.PostConsensusSyncCommitteeContributionMsg(ks.Shares[2], 2, ks),
+			testingutils.PostConsensusSyncCommitteeContributionMsg(ks.Shares[3], 3, ks),
```

Comment:
should we test with messages other than synccommitteecontrib?

Reply:
maybe...
when I look at this now not sure why this test is part of the PR (it is something Alon added)...
The PR should concentrate on preconsensus...

nevertheless I am adding you point in #264